test/parallel/test-icu-data-dir.js

```diff
@@ -8,7 +8,7 @@ const { spawnSync } = require('child_process');
 
 const expected =
     'could not initialize ICU (check NODE_ICU_DATA or ' +
-    '--icu-data-dir parameters)' + (common.isWindows ? '\r\n' : '\n');
+    `--icu-data-dir parameters)${common.isWindows ? '\r\n' : '\n'}`;
```

Comment:
While you're here, could you replace `${common.isWindows ? '\r\n' : '\n'}`` with `os.EOL` (you'll have to `const os = require('os')` after line 2 as well)?